frontend/contact/index.html

```diff
@@ -125,6 +123,21 @@ <h3>My domain (or one of its subdomains) is broken due to preloading, and waitin
 
   <section id="other">
     <h2><a class="hash-link" href="#other">Other questions</a></h2>
+
+    <h3>Customer support for my service provider told me to contact you for support</h3>
```

Comment:
nit: "My service provider told me to contact you for support" instead? (Otherwise "support" feels a little confusing to me since it's used in two forms in the same sentence.)

Reply:
That sounds better - thanks!